tests/model/DataExtensionTest.php

```diff
@@ -159,13 +159,20 @@ public function testExtensionCanBeAppliedToDataObject() {
 		$do = new DataObject();
 		$mo = new DataExtensionTest_MyObject();
 
-		$this->assertTrue($do->hasMethod('testMethodApplied'));
 		$this->assertTrue($mo->hasMethod('testMethodApplied'));
+		$this->assertTrue($do->hasMethod('testMethodApplied'));
 
 		$this->assertEquals("hello world", $mo->testMethodApplied());
 		$this->assertEquals("hello world", $do->testMethodApplied());
 	}
 
+	public function testExtensionAllMethodNamesHasOwner() {
+		/** @var DataExtensionTest_MyObject $do */
+		$do = DataExtensionTest_MyObject::create();
+
+		$this->assertTrue($do->hasMethod('getTestValueWithDataExtensionTest_MyObject'));
```

Comment:
Heh, it took me some pondering to make a valid test, but in the end, I figured "All that needs testing, is if the owner is set". Thus, just checking and add some syntactic sugar to it was all that it needed :)